src/pages/AppLogin.vue

```diff
@@ -71,101 +83,85 @@ onMounted(async () => {
-                @click.stop="launchLogin('google')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/google-sso.svg"
-                  />
-                  <span class="body-1">Google</span>
-                </div>
-              </v-card-button>
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('github')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/github-sso.svg"
-                  />
-                  <span class="body-1">Github</span>
-                </div>
-              </v-card-button>
-            </div>
-            <div
-              class="flex wrap"
-              style="gap: 1em; justify-content: space-between; margin-top: 2em"
-            >
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('twitch')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/twitch-sso.svg"
-                  />
-                  <span class="body-1">Twitch</span>
-                </div>
-              </v-card-button>
-              <v-card-button
-                class="sso-button"
-                @click.stop="launchLogin('discord')"
-              >
-                <div class="flex" style="align-items: center; padding: 0.2em 0">
-                  <img
-                    style="margin-right: 1em"
-                    src="@/assets/discord-sso.svg"
-                  />
-                  <span class="body-1">Discord</span>
-                </div>
-              </v-card-button>
-            </div>
+      <LandingDescriptor />
+      <section class="signin-section">
+        <div class="signin-container">
+          <img
+            src="@/assets/logo.svg"
+            alt="Arcana Logo"
+            style="margin: auto; margin-bottom: 2em"
+            class="laptop-remove"
+          />
+          <div>
+            <h1>Welcome</h1>
+            <h5 class="login-description">
+              We’ll email you a magic link for a password-free sign in.
+            </h5>
+          </div>
+          <div class="passwordless-container flex column flex-center">
+            <VTextField
+              v-model.trim="email"
+              label="Email"
+              class="passwordless-email"
+              message="Invalid Email"
+              :message-type="hasValidEmail ? '' : 'error'"
+              @update:model-value="validateEmail"
+              @keyup.enter="
+                email && hasValidEmail ? launchLogin('passwordless') : void 0
```

Comment:
Can we move this logic out of the template?

Reply:
Removed @keyup.enter because we have wrapped these textfield and button in a form element and it will automatically handle the enter as submit now

https://github.com/arcana-network/developer-dashboard/pull/121/commits/7b00c57f2704f07bd58178b721d4f56fff015592